.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,47 @@
+<!-- ## Which branch should I use for my PR?
+
+Assuming that:
+
+a = current major release
+b = current minor release
+c = future major release
+
+* a.x for any features and enhancements (e.g. 4.x)
+* a.b for any bug fixes (e.g. 4.0, 4.1, 4.2)
+* c.x for any features, enhancements or bug fixes with backward compatibility breaking changes (e.g. 5.x) -->
+
 | Q                                      | A
 | -------------------------------------- | ---
-| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) <!-- see below -->
-| Bug fix?                               | yes/no
-| New feature?                           | yes/no
-| Deprecations?                          | yes/no
-| BC breaks?                             | yes/no
-| Automated tests included?              | yes/no
+| Bug fix? (use the a.b branch)          | [ ]
+| New feature/enhancement? (use the a.x branch)      | [ ]
+| Deprecations?                          | [ ]
+| BC breaks? (use the c.x branch)        | [ ]
+| Automated tests included?              | [ ] <!-- All PRs must maintain or improve code coverage -->
 | Related user documentation PR URL      | mautic/mautic-documentation#... <!-- required for new features -->
 | Related developer documentation PR URL | mautic/developer-documentation#... <!-- required for developer-facing changes -->
 | Issue(s) addressed                     | Fixes #... <!-- prefix each issue number with "Fixes #", no need to create an issue if none exists, explain below instead -->
 
 <!--
-Additionally (see https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests#step-5-work-on-your-pull-request):
+Additionally (see https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests#work-on-your-pull-request):
  - Always add tests and ensure they pass.
  - Bug fixes must be submitted against the lowest maintained branch where they apply
    (lowest branches are regularly merged to upper ones so they get the fixes too.)
- - Features and deprecations must be submitted against the "features" branch.
+ - Features and deprecations must be submitted against the "4.x" branch.
 -->
 
-<!--
-Please write a short README for your feature/bugfix. This will help people understand your PR and what it aims to do.
--->
 #### Description:
 
 <!--
-If you are fixing a bug and if there is no linked issue already, please provide steps to reproduce the issue here.
+Please write a short README for your feature/bugfix. This will help people understand your PR and what it aims to do. If you are fixing a bug and if there is no linked issue already, please provide steps to reproduce the issue here.
 -->
 
 #### Steps to test this PR:
+
+<!--
+This part is really important. If you want your PR to be merged, take the time to write very clear, annotated and step by step test instructions. Do not assume any previous knowledge - testers may not be developers.
```

Comment:
Thanks for adding this comment! Let's monitor the quality of PR testing steps. If needed in the future, we could provide a URL here with examples of clear PR testing steps.